rerun_actions.go

```diff
@@ -13,7 +13,12 @@ import (
 	"golang.org/x/oauth2"
 )
 
-const testAll = "__all"
+const (
+	testAll             = "__all"
+	canTestLabel        = "ok-to-test"
+	retestCommand       = "retest"
+	testWorkflowCommand = "test-action"
```

Comment:
This could also be `/test-job ...` as well? I'm just getting up to speed in the GH action world, so I'm not sure what term is the most applicable.

Reply:
What about `/rerun-all` and `/rerun-workflow <workflow name>`, to make the commands more generic (since the workflows might not be tests)?